docs/tutorials/fundamentals/viewer.md

```diff
@@ -76,81 +57,89 @@ import napari
+
+* **File** contains the options shown in the graphic below:  
+![image: File Menu](../assets/tutorials/main-menu-file.png)  
 
-The canvas is in the center of the viewer and contains the visual display
-of the data passed to **napari**, including images, point, shapes, and our other
-supported data types. Under the hood the canvas is a `vispy.scene.SceneCanvas`
-object which has built-in support for features such as zooming and panning. As
-`vispy` uses `OpenGL` and your graphics card, panning and zooming are highly
-performant. You can also return to the original zoom level by clicking the
-`home` button in the viewer buttons panel.
+    All the options on the **File** menu are self-explanatory except **Preferences**. **Preferences** has the options shown below. To learn more about the **Preferences** menu, visit this [tutorial](https://napari.org/stable/guides/preferences.html).  
+    ![image: Preferences sub-menu](../assets/tutorials/main-menu-file-sub-menu-preferences.png)  
+    </br>
+    
 
-+++
+* **View** contains the options shown in the graphic below:  
+![image: View Menu](../assets/tutorials/main-menu-view.png)    
+    </br>
+  
 
-![image: pan and zoom with napari](../assets/tutorials/viewer_pan_zoom.webm)
+* **Window** contains the options shown in the graphic below:   
+![image: Window Menu](../assets/tutorials/main-menu-window.png)    
+    </br>
 
-+++
+* **Plugins** contains the options shown in the graphic below:   
+![image: Plugins Menu](../assets/tutorials/main-menu-plugins.png)    
+    </br>
+
+* **Help** contains the options shown in the graphic below:   
+![image: Help Menu](../assets/tutorials/main-menu-help.png)    
+    </br>
+
+### Canvas
+
+The **canvas** is in the center of the viewer and contains the visual display of the data passed to **napari**, including `images`, `point`, `shapes`, and other supported data types. Under the hood, the canvas is a `vispy.scene.SceneCanvas` object which has built-in support for features such as zooming and panning. As `vispy` uses `OpenGL` and your graphics card, panning and zooming are highly performant. You can return to the original zoom level by clicking the `home` button in the viewer buttons panel.
+
+![image: pan and zoom with napari](../assets/tutorials/viewer_pan_zoom.webm) 
 
 (layer_list)=
 ### Layer list
 
-One of the basic **napari** objects are layers. There are different layer types
-for `Image`, `Points`, `Shapes`, and other basic data types. They can be added
-to the viewer either programmatically or through the GUI. Once added they start
-to populate the layer list located on the bottom lefthand side of the main
-canvas.
+Layers are one of the basic napari objects. There are different layer types for `image`, `points`, `shapes`, and other data types. They can be added to the viewer either programmatically or through the GUI. Once added, they  populate the layer list located on the bottom left side of the canvas.
```

Comment:
Now that we are referring to specific layer types, I think they should be capitalized to be consistent with other docs. Scroll down to Layer controls for example.